packages/common/src/adapters/user.ts

```diff
@@ -112,3 +115,28 @@ export const userManagerFromSDK = (
 
 export const userManagerListFromSDK = (input?: full.UserManager[]) =>
   input ? input.map((d) => userManagerFromSDK(d)).filter(removeNullable) : []
+
+export const accountFromSDK = (
+  input: full.AccountFull
+): AccountUserMetadata | undefined => {
+  const user = userMetadataFromSDK(input.user)
+  if (!user) {
+    return undefined
+  }
+  return {
```

Comment:
not sure how large or nested this data is, but we could perhaps spread the repeated stuff to DRY up 

```ts
const userExtras = {
  playlists: transformAndCleanList(input.playlists, accountCollectionFromSDK),
  playlist_library: playlistLibraryFromSDK(input.playlistLibrary) ?? null,
  track_save_count: input.trackSaveCount
}

return {
  user: {
    ...user,
    ...userExtras
  },
  ...userExtras
}
```

Reply:
Oh yeah good point. I blindly copied the props into the user object because we temporarily need them to live there until I can split them out. But we definitely don't need to run transformations on it twice.